src/pages/Cohort/ActionButtons.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { ButtonSize } from '../../utils/types'
+import Button from '../../components/ui/Button'
+
+const ActionButtons = () => {
+  return (
+    <div className='flex items-center justify-center space-x-2 w-full h-full'>
+      <Button size={ButtonSize.Small} outlined className='text-xs'>
```

Comment:
If `text-xs` is put on the above div, you'd not need to pass `className` property in the button.